collectors/chain_collector.go

```diff
@@ -3,7 +3,7 @@ package collectors
 import (
 	"context"
 
-	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightninglabs/lndclient"
```

Comment:
Hmm, will switching over to using `lndclient` actually cause more issues than it solves? Main thing I'm weary of is the need to update `lndclient` to expose some new functionality, rather than just updating the `lnd` package commit, then immediately having access to the new features.

Reply:
I haven't found that using `lndclient` has been much of a blocker for `loop`/`faraday`, the `lndclient` PRs are pretty straightforward to review and  I do think we get some degree of increasing returns when we add endpoints to `lndclient` because they're then available for other projects.  We can do this change without a switchover ofc, so happy to remove it if you think it'll slow dev too much.

Reply:
Ok np, if y'all don't think it slows things down, then I'm ok with the switch.